### PR TITLE
Added police shooting from overpass in Louisville

### PR DESCRIPTION
**Pull request by kaznatcheev**

### CODE REVIEW

**Comment by BonkyMaroo** [comment]
Date is likely the day before (May 31st) since the video was posted to Twitter at 9:52am. It's night in the video, and the guy narrating says that these are people going home from the protest.

**Comment by valkyrie9812** [comment]
fixed the date. Anything else we should change?

File: reports/Kentucky.md [hunk 1]
```diff
@@ -34,6 +34,14 @@ A young woman was injured by a rubber bullet she took to the head.
 * [Video](https://twitter.com/shannynsharyse/status/1267015577266249728)
 * [Photo of Victim](https://twitter.com/shannynsharyse/status/1266631722239766528)
 
+### Police shoot at cars in traffic from overpass | May 31st
+
+Several police officers shoot from an overpass at cars stuck in traffic. Bystander reports at least one car window broken.
```
Comment (maxspencer) [inline reports/Kentucky.md:39]: Are we able to be specific about what they shot? Gas canisters? Less-lethal bullets?

Comment (maxspencer) [reply reports/Kentucky.md:39]: I'm going to merge this as-is now, but feel free to raise another PR if you want to address this.

**Pull request merged**